drafts/2018-11-20-this-week-in-rust.md

```diff
@@ -158,6 +158,8 @@ Email the [Rust Community Team][community] for access.
 * [Software Infrastructure Engineer - Engines at Blue Origin, Kent, US](https://careers-blueorigin.icims.com/jobs/3247/software-infrastructure-engineer---engines/job).
 * [Rust web developer at Impero, Denmark/remote](https://impero.com/job/full-stack-web-developer-rust/)
 * [Rust Developer at Parity, Berlin, DE](https://paritytech.io/jobs/).
+* [Embedded operating system developer, Karlsruhe, DE](https://www.pse.kit.edu/karriere/joboffer.php?id=2093&language=en)
+* [Student research assistant (embedded), Karlsruhe, DE](mailto:oliver.scherer@kit.edu)
```

Comment:
Is there a webpage to link to (or even a tweet with more information on how to contact will do).

Reply:
there is now